docs/src/use/configure/migration-guide.md

```diff
@@ -330,6 +330,8 @@ export default [
 ];
 ```
 
+Note that for this to work or if you get `Global "AudioWorkletGlobalScope " has leading or trailing whitespace`, you'll need to install the latest `globals` package.
```

Comment:
This is way too specific an error message to include here. We can also format this a bit nicer.

```suggestion
::: tip
You'll need to install the `globals` package from npm for this example to work. It is not automatically installed by ESLint.
:::
```